test/web3-functions/mainnet/xchain-oracle-ticker.test.ts

```diff
@@ -0,0 +1,199 @@
+import hre from 'hardhat'
+import { expect } from 'chai'
+import { Contract } from '@ethersproject/contracts'
+import { Web3FunctionResultCallData } from '@gelatonetwork/web3-functions-sdk/*'
+import { Web3FunctionHardhat } from '@gelatonetwork/web3-functions-sdk/hardhat-plugin'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
```

Comment:
Can you confirm this file isn't duplicated? Not sure why its in the diff

Reply:
Yes, it was. Fixed.